addon/HomepageStudy.jsm

```diff
@@ -11,13 +11,18 @@ const {classes:Cc, interfaces: Ci, utils: Cu} = Components;
 const EXPORTED_SYMBOLS = ["HomepageStudy"];
 
 Cu.import("resource://gre/modules/Services.jsm");
+Cu.import("resource://gre/modules/Console.jsm");
+Cu.import("resource://gre/modules/FileUtils.jsm");
 Cu.import("resource://gre/modules/Log.jsm");
 
-const RAPPOR_PATH = `chrome://shield-study-rappor/content/TelemetryRappor.jsm`;
-const { TelemetryRappor } = Cu.import(RAPPOR_PATH, {});
-
 const PREF_HOMEPAGE = "browser.startup.homepage";
 
+const TELEMETRY_RAPPOR_PATH = `chrome://shield-study-rappor/content/TelemetryRappor.jsm`;
```

Comment:
Why did you move these? No need to, it just clutters the history :)

Reply:
Done